aliases/names.go

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
+
+package aliases
+
+import (
+	"fmt"
+)
+
+// IsValid decides weather an alias is okay for use or not.
+// The room spec defines it as _labels valid under RFC 1035_ ( https://ssb-ngi-pointer.github.io/rooms2/#alias-string )
+// but that can be mostly any string since DNS is a 8bit binary protocol,
+// as long as it's shorter then 63 charachters.
+//
+// Right now it's pretty basic set of charachters (a-z, A-Z, 0-9).
```

Comment:
```suggestion
// Right now it's pretty basic set of characters (a-z, A-Z, 0-9).
```

Reply:
Right.. thanks.

I wanted to copy my mini-rant about _which letters can be in a subdomain_ into an issue as well and link it there, too.